web/src/components/core/Page.jsx

```diff
@@ -23,78 +23,130 @@ import React from "react";
 import { useNavigate } from "react-router-dom";
 import { Button } from "@patternfly/react-core";
 
-import { noop } from "~/utils";
-import { Icon, Title, PageIcon, MainActions } from "~/components/layout";
+import logoUrl from "~/assets/suse-horizontal-logo.svg";
+
+import { _ } from "~/i18n";
+import { partition } from "~/utils";
+import { Icon } from "~/components/layout";
+import { If } from "~/components/core";
+
+/**
+ * Wrapper component for holding Page actions
+ *
+ * Useful and required for placing the components to be used as Page actions, usually a
+ * Page.Action or PF/Button
+ *
+ * @see Page examples.
+ *
+ * @param {React.ReactNode} [props.children] - a collection of Action components
+ */
+const Actions = ({ children }) => <>{children}</>;
+
+/**
+ * A convenient component representing a Page action
+ *
+ * Built on top of {@link https://www.patternfly.org/components/button PF/Button}
+ *
+ * @see Page examples.
+ *
+ * @param {React.ReactNode} props.children - content of the action
+ * @param {object} [props] - PF/Button props, see {@link https://www.patternfly.org/components/button#props}
+ */
+const Action = ({ children, navigateTo, onClick, ...props }) => {
+  const navigate = useNavigate();
+
+  props.onClick = () => {
+    if (typeof onClick === "function") onClick();
+    if (navigateTo) navigate(navigateTo);
+  };
+
+  if (!props.size) props.size = "lg";
+
+  return <Button { ...props }>{children}</Button>;
+};
+
+/**
+ * Simple action for navigating back
+ *
+ * @note it will be used by default if a page is mounted without actions
+ *
+ * TODO: Explain below note better
+ * @note that we cannot use navigate("..") because our routes are all nested in
+ * the root.
+ *
+ * @param {React.ReactNode} props.children - content of the action
```

Comment:
leftover params?

Reply:
Yes, I realized a few minutes ago too, while doing a merge into #886 

Will take care of it ASAP.

Reply:
Actually something weird happens there (lines switched or so) plus the new props were not documented. As said above, I'll fix it as part of another PR. Or maybe a tiny one for this kind of things, to not forget about them. But tomorrow or next week.